okta/framework/ApiClient.py

```diff
@@ -24,16 +24,23 @@ def __init__(self, *args, **kwargs):
         elif len(args) > 1 and args[1]:
             self.api_token = args[1]
         else:
-            raise ValueError('An api_token must be provied to create an ApiClient')
+            # raise ValueError('An api_token must be provied to create an ApiClient')
+            pass
 
         self.api_version = 1
         self.max_attempts = 4
 
-        self.headers = {
-            'Accept': 'application/json',
-            'Content-Type': 'application/json',
-            'Authorization': 'SSWS ' + self.api_token
-        }
+        if kwargs['pathname'] == '/api/v1/authn':
```

Comment:
This only matches `/api/v1/authn`, so any other nested authn apis (like enrolling in a factor with `/api/v1/authn/factors`) would attempt to send the `api_token`